attempt to fix urls and view.py

YazanSalem opened the pull request:
I based this off of Waj's changes when he added his accountsettings.html to fix editCourse and editUser, please let me know if I done this incorrectly

YazanSalem committed:
attempt to fix urls and view.py

YazanSalem committed:
need to delete those 2 lines for program to run

BMHebert commented on CS361Project/urls.py:
```diff
@@ -0,0 +1,30 @@
+
+from django.contrib import admin
+from django.urls import path
+from TAScheduler.views import Login, Home, CreateUser, CreateCourse, AccountSettings, EditUser, EditCourse
```
```suggestion
from TAScheduler.views import *
```
The asterisk imports all the classes from views so they don't have to be all typed out.

BMHebert reviewed (approved):
These changes all look great. Thanks for doing this.

BMHebert committed:
Merge branch 'main' into Communcations

BMHebert commented on TAScheduler/views.py:
```diff
@@ -1,10 +1,26 @@
 from .models import UserProfile, Course, Lab
-from TAScheduler.Managment.UserManagement import UserManagement
+from TAScheduler.Management.UserManagement import UserManagement
 
```
```suggestion
from TAScheduler.Management.CourseManagement import CourseManagement

```

BMHebert committed:
Update TAScheduler/views.py

Added back in the CourseManagement import since it had a typo before and that's why it was making the program fail. It will need to be used in the future, so it's just nice to have it there now.
